pdns/pkcs11signers.cc

```diff
@@ -289,8 +295,39 @@ class Pkcs11Token {
       }
     }
 
+    unsigned int ecparam2bits(const std::string& obj) {
+      // if we can use some library to parse the EC parameters, better use it.
+      // otherwise fall back to using hardcoded primev256 and secp384r1
+#ifdef HAVE_LIBCRYPTO_ECDSA
+      EC_KEY *key = NULL;
+      BIGNUM *order;
+      unsigned int bits = 0;
+      unsigned char data[obj.size()];
+      const unsigned char *in = data;
+      obj.copy((char*)in, obj.size());
```

Comment:
I'm very likely missing something but I don't see why we need to copy the content of `obj`?